cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,14 @@
         "AuditTable": {
             "Description": "Name of the audit dynamo table",
             "Type": "String"
+        },
+        "AlertWebhook": {
+          "Description": "Where CloudWatch alerts are sent",
+          "Type": "String"
+        },
+        "AlertActive": {
```

Comment:
I don't know. I was copying how the grid does it.

Happy to remove it if you want?